websites/M/MoodlePresence/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+    clientId: "821059536768335902"
+}),
+    timebrowsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const pathName = document.location.pathname,
+        pages = pathName.split('/').filter(p => p),
+        data: PresenceData = {
+            largeImageKey: "largeimage",
+            smallImageText: 'Reading . . .',
```

Comment:
Changed "Reading . . ." to "Reading..."

Reply:
why tho

Reply:
Dont sert a smallIamgeText if you dont have a smallImage...